presto-spark-base/src/main/java/com/facebook/presto/spark/execution/shuffle/PrestoSparkLocalShuffleInfoTranslator.java

```diff
@@ -51,16 +52,18 @@ public PrestoSparkLocalShuffleInfoTranslator(
     }
 
     @Override
-    public PrestoSparkLocalShuffleWriteInfo createShuffleWriteInfo(PrestoSparkShuffleWriteDescriptor writeDescriptor)
+    public PrestoSparkLocalShuffleWriteInfo createShuffleWriteInfo(Session session, PrestoSparkShuffleWriteDescriptor writeDescriptor)
```

Comment:
Instead of passing the `session` object, can we just pass in what is exactly needed - `QueryId` ?